src/clj/witan/send/params.clj

```diff
@@ -187,6 +180,18 @@
     (continue-for-latter-ays params academic-years)))
 
 
+(defn mover-alpha-priors
+  "Calculate uniform prior across all valid transitions from state"
+  [need-setting valid-transitions valid-settings observations-per-ay ay]
+  (let [[need setting] (s/split-need-setting need-setting)
+        allowed-settings (->> (intersection (set (get valid-settings [ay need]))
+                                            (set (get valid-transitions setting)))
+                              (remove #{setting}))
+        prior (->> (zipmap allowed-settings (repeat (/ 1.0 (count allowed-settings))))
+                   (merge-with + (get observations-per-ay ay)))]
+    (zipmap (map #(s/join-need-setting need %) (keys prior)) (vals prior))))
```

Comment:
This looks to be missing the normalisation ie. the the division through of the counts and priors by the total.

Reply:
I think this would be a good place to write some tests around the original function and prove out our refactor doesn't change the result.  Tests around the whole function to start with and more specific tests around the prior calculation.

Reply:
As to whether it should be normalised is another question entirely....but we have been to date.

Reply:
My guess is this is also why the validation broke.